src/backend/distributed/commands/multi_copy.c

```diff
@@ -987,25 +863,13 @@ OpenCopyTransactions(CopyStmt *copyStatement, ShardConnections *shardConnections
 		ShardPlacement *placement = (ShardPlacement *) lfirst(placementCell);
 		char *nodeName = placement->nodeName;
 		int nodePort = placement->nodePort;
-		WorkerNode *workerNode = FindWorkerNode(nodeName, nodePort);
-		int workerGroupId = 0;
 		char *nodeUser = CurrentUserName();
-		PGconn *connection = ConnectToNode(nodeName, nodePort, nodeUser);
-
-		TransactionConnection *transactionConnection = NULL;
+		MultiConnection *connection = NULL;
+		uint32 connectionFlags = SESSION_LIFESPAN | FOR_DML;
```

Comment:
May have missed something, but I don't think we want to use SESSION_LIFESPAN by default.